.azure-pipelines/generation-pipeline.yml

```diff
@@ -154,6 +154,7 @@ variables:
   kiotaDirectory: '$(Build.SourcesDirectory)/kiota'
   typewriterExecutable: '$(typewriterDirectory)/Typewriter'
   scriptsDirectory: '$(Build.SourcesDirectory)/MSGraph-SDK-Code-Generator/scripts'
+  conversionSettingsDirectory: '$(Build.SourcesDirectory)/MSGraph-SDK-Code-Generator/conversion-settings'
```

Comment:
this needs to be updated as well, And the powershell script could rely on this variable instead of building the path itself

Reply:
This surprisingly failed in my tests. Having it like this was the only way it passed

Reply:
right because variables don't survive between jobs, thanks for fixing the original one